website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -19,6 +19,7 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 \* The official release date for this new format of release notes is May 15th, 2024. Historical release notes for prior dates may not reflect all available features released earlier this year or their tenancy availability.
 
 ## August 2024
+- **Fix:** Fixed an issue in Explorer where navigating to a consumer project from a public node resulted in displaying a random public model rather than the original selection. 
```

Comment:
sugg: use proper noun of explorer and link to the docs page 

```suggestion
- **Fix:** Fixed an issue in [dbt Explorer](/docs/collaborate/explore-projects) where navigating to a consumer project from a public node resulted in displaying a random public model rather than the original selection. 
```